docs/09-dev-guide.md

```diff
@@ -147,33 +133,30 @@ Run this occasionally until it reports that there are 440 million records in the
 
 6. Publish the performance statistics.
 
-Record the ingest rate in the [performance figures](12-performance-test.md) documentation.
-
-A compaction job will now be created (as this is done using periodically scheduled EventBridge rule
-it may take a little while for the job to be created). Once this has been done use the following
-command to monitor the status of the compaction job:
-
+Record the ingest and compaction rate in the [performance figures](12-performance-test.md) documentation.
 ```bash
+# Ingest performance figures can be found by running the following reports
+./scripts/utility/ingestTaskStatusReport.sh ${ID} standard -a
+./scripts/utility/ingestJobStatusReport.sh ${ID} system-test standard -a
+
+# Compaction performance figures can be found by running the following reports
+./scripts/utility/compactionTaskStatusReport.sh ${ID} standard -a
 ./scripts/utility/compactionJobStatusReport.sh ${ID} system-test standard -a
 ```
 
-Record the compaction rate in the [performance figures](12-performance-test.md) documentation. Commit these
-changes.
-
-The following tests can be used as a quick check that all is working correctly. They are not intended to
-fully test all aspects of the system. Any changes made by pull requests should be tested by doing a system
-test deployment on AWS if they are likely to either affect performance or involve changes to the way
-the system is deployed to AWS.
+The following tests can be used as a quick check that all is working correctly. They are not intended to fully test 
+all aspects of the system. Any changes made by pull requests should be tested by doing a system test deployment on AWS 
+if they are likely to either affect performance or involve changes to the way the system is deployed to AWS.
 
 7. Test a simple query:
 
 ```bash
 ./scripts/utility/query.sh ${ID}
```

Comment:
The compaction performance test doesn't deploy the query stack, so the query steps will need to be done with the deployAll test, or by adding the stack in the compaction performance test properties.

This is a bit awkward but we'll add functional system tests for this soon.